config.gpu.yaml

```diff
@@ -24,19 +24,18 @@ vectordb:
     # url: http://localhost:8080
 
 embedding:
+  llama:
```

Comment:
not sure in the case of embedding mode, but does `n_gpu_layers: -1` make a difference?